scripts/channelcommands.js

```diff
@@ -231,33 +234,33 @@ exports.handleCommand = function (src, command, commandData, tar, channel) {
             if (isNaN(part) || part > topic.length || pospart < 0) {
                 channelbot.sendMessage(src, "Parts must be a number from 1 to " + topic.length + "!", channel);
                 return;
-            } else if (duplicates.indexOf(part) !== -1) {
+            }
+            if (duplicates.indexOf(part) !== -1) {
                 channelbot.sendMessage(src, "You can't remove part " + part + " twice!", channel);
                 return;
             }
             duplicates.push(part);
         }
         // Sort by largest numbers first to avoid interfering with earlier parts after removal
-        parts.sort(function(a, b) { return b - a; }).forEach(function(part) {
+        parts.sort(function (a, b) { return b - a; }).forEach(function (part) {
             topic.splice(part - 1, 1);
         });
         SESSION.channels(channel).setTopic(src, topic.join(Config.topic_delimiter));
         return;
-    }   
+    }
     if (command === "topicparts") {
         var topic = SESSION.channels(channel).topic;
-        if (typeof topic == "string" && topic !== "") {
+        if (typeof topic === "string" && topic !== "") {
             var i = 0;
-            topic = topic.split(" | ").map(function(part) { 
+            topic = topic.split(" | ").map(function (part) {

```

Comment:
This could probably use Config.topic_delimiter instead of " | "  (same with line 257) =D